DataLink.tex

```diff
@@ -1325,25 +1336,34 @@ \section{Changes}
 \subsection{DataLink-1.1}
 
 \begin{itemize}
-\item allow optional columns to contain values when the row (link) has an error\_message
+\item allow optional columns to contain values when the row (link) has an error\_message 
+(see Section \ref{sec:errorMessage})
 \item added optional local\_semantics to identify corresponding rows 
-	for different IDs in the same service
-\item relax content-type usage to allow any valid VOTable MIME type
-\item INFO element with standardID mandatory in \blinks\ response
+	for different IDs in the same service (see Section \ref{sec:localSemantics})
+\item relax content-type usage to allow any valid VOTable MIME type (see Section 
+  \ref{sec:mime})
```

Comment:
The reference here should actually be to sec 3.3 not 3.1 (sec:successfulRequests).

Reply:
Thanks for the correction, change made.